pygenometracks/readBed.py

```diff
@@ -270,6 +270,36 @@ def get_bed_interval(self, bed_line):
             "Start position larger or equal than end" \
             " for line #{}:\n{}\n".format(self.line_number,
                                           bed_line)
+        if len(line_values) == 12:
```

Comment:
maybe this can be an extra function `check_bed12` to keep the code more concise?

Reply:
Yes, good idea, thanks.